tests/unit/utils/schema-version-check.spec.js

```diff
@@ -75,4 +79,27 @@ describe('schema-version-check', () => {
     emitWarningForSchemaVersionMismatch();
     expect(warnSpy).toHaveBeenCalledTimes(0);
   });
+
+  describe('compareVersions', () => {
+    it.each([
+      ['1.1.1', '1.1.10', -1],
+      ['1.1.1', '1.2.10', -1],
+      ['1.1.20', '1.2.10', -1],
+      ['1.1.1', '2.1.1', -1],
+      ['1.1', '1.1.0', -1],
```

Comment:
Shouldn't this return `0`? I think it's unlikely for us to see this kind of comparison with realistic data, but it still seems odd to me that we would treat `1.1` as being less than `1.1.0`.